.github/workflows/bazel.yml

```diff
@@ -0,0 +1,48 @@
+name: bazel
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'Sources/**/*.[ch]'
+      - 'Sources/**/*.swift'
+      - 'Tests/**/*.swift'
+      - 'Tests/**/*.ya?ml'
+      - '!Tests/LinuxMain.swift'
+      - '**/BUILD'
+      - 'WORKSPACE'
+  pull_request:
+    paths:
+      - 'Sources/**/*.[ch]'
+      - 'Sources/**/*.swift'
+      - 'Tests/**/*.swift'
+      - 'Tests/**/*.ya?ml'
+      - '!Tests/LinuxMain.swift'
+      - '**/BUILD'
+      - 'WORKSPACE'
+
+jobs:
+  bazel:
+    runs-on: macos-latest
```

Comment:
Would it be possible to run these on Linux too?

Reply:
Yes will add that